documentation/incremental_settings.md

```diff
@@ -0,0 +1,85 @@
+# Incremental Settings for Ansible Values
+
+In some cases in Ansible, you provide most or all of the pertient attributes
+of the managed object.  For example, for the `file` module, you provide the
+content, permissions, ownership, and other attributes of the file to be
+managed. In other cases, you have to specify how to change the *incremental*
```

Comment:
I would write "how to *incrementally* change the state of the system" instead of "how to change the *incremental* state of the system". Incremental state to me sounds like an oxymoron (state is not the increment, but the thing on top of which you apply the increment and the result (another state). I.e. state is an integral, increment a derivative. (Actually a sum and a difference, but IKWYM)).

Reply:
Fixed in https://github.com/linux-system-roles/linux-system-roles.github.io/pull/21/commits/373b5ac990e225b3ac11431fef1154a65e047475